lib/plugins/aws/package/compile/events/apiGateway/lib/requestValidator.test.js

```diff
@@ -0,0 +1,416 @@
+'use strict';
+
+const chai = require('chai');
+const runServerless = require('../../../../../../../../tests/utils/run-serverless');
+const fixtures = require('../../../../../../../../tests/fixtures');
+
+const expect = chai.expect;
+chai.use(require('chai-as-promised'));
+
+describe('#compileRequestValidators()', () => {
+  let serverlessInstance;
+
+  before(() => {
+    return Promise.resolve(
+      runServerless({
+        cwd: fixtures.map.requestSchema,
+        cliArgs: ['package'],
+      })
+    ).then(serverless => (serverlessInstance = serverless));
```

Comment:
`runServerless` got improved over course of time, and now it needs to be setup as: `({ serverless }) => (serverlessInstance = serverless)`